monero-outreach-docs/translations/es/merchants/how_to_accept_monero_en.md

```diff
@@ -0,0 +1,85 @@
+
+Muchos dueños de negocios están familiarizados con procesadores de pago de crédito/débito, terceros que te alquilan un terminal, quienes te meten en su red de pagos y te cargan con comisiones por cada pago y cada mes. Un procesador de pagos de Monero puede funcionar así, o puede ser tu propio procesador de pagos con un poco de software y sin tener que pagar una tarifa.
+
+Como el software Monero es de código abierto, puedes crear tu propia pasarela de pago desde cero (eso es genial), pero las integraciones de Monero han hecho ya el trabajo por ti y crearon bibliotecas y complementos que hacen todo el trabajo duro en [github.com/monero-integrations](https://github.com/monero-integrations).
+
+Alternativamente, un procesador de pagos Monero de terceros tendrá software y aplicaciones e interfaces empaquetados que simplifican la configuración. También tendrán interfaces y herramientas para que usted administre su cuenta y un soporte técnico y de atención al cliente. Hecha un vistazo a nuestra guía de procesadores de pago de Monero para ayudarte a decidir que configuración te viene mejor.
+
+### _LeMonero.com_
+
+Nuestra tienda de limonada está encantándole a la gente y nos hemos expandido a las ventas en línea y a más locales. La buena noticia es que no necesitamos cambiar nuestro procesador de pagos, solo necesitamos usarlo más.
+
+La integración del carrito de compras es fácil en comparación con las transacciones físicas en tiempo real. Ya sea que estés utilizando un procesador de pagos de código abierto o un servicio de terceros, los complementos para WooCommerce, Shopify, etc. están disponibles y son fáciles de instalar.
+
+Ahora podemos tener liquidaciónes automáticas en fiat. Como puesto de limonada, mantenemos nuestros pagos en XMR; cuando éramos una tienda, liquidábamos algunos cuando el cambio era favorable. Pero nuestras necesidades de flujo de caja se han intensificado. Necesitamos un cambio de XMR a fiat predecible, incluso si eso significa tarifas adicionales o un momento desfavorable.
+
+La mayoría de los procesadores de pagos de terceros proporcionarán conversión a fiat automática con los intercambios de sus socios por una tarifa. Pueden haber tarifas de transacción o retiro adicionales dependiendo de los proveedores, pero si nos conviene, puede valer la pena.
+
+Con un procesador de pagos de código abierto eres libre de desarrollar tu propia solución usando la API de tu exchange preferido con llamadas al monero-wallet-rpc. Esto va más allá de el objetivo de este tutorial, pero puedes encontrar más sobre ello en [getmonero.org/resources/developer-guides/wallet-rpc.html](https://www.getmonero.org/resources/developer-guides/wallet-rpc.html).
+
+### _Aprende más_
+
+- Cómo aceptar Monero con las billeteras oficiales: [getmonero.org/get-started/accepting](https://www.getmonero.org/get-started/accepting/)
```

Comment:
Cómo aceptar Monero con **los monederos** oficiales: [**getmonero.org/es/get-started/accepting**](https://www.getmonero.org/es/get-started/accepting/index.html)

Missed this one too, **"los monederos"** and the link for the Spanish getmonero site -> https://www.getmonero.org/es/get-started/accepting/index.html